frontend/src/Map/style/scripts/build.js

```diff
@@ -0,0 +1,63 @@
+// Forked from https://github.com/klokantech/gl-style-package-spec/blob/master/task/deploy.js
+// and https://github.com/klokantech/gl-style-package-spec/blob/master/task/transform.js
+
```

Comment:
can you explain why these are included?

Reply:
oh, is this for sprites (as you mentioned in the PR comment)? Might be nice to have a commented line on why we included this. Guess a note in the commit history would work too. Both?